### PR TITLE
Tbolt/4491 medicaid business areas bug

### LINKED ISSUE #4491: [Bug] MMIS Medicaid Business Areas missing field data
### Steps to reproduce

1. Inspect the website to pull up developer tools while a new APD is created
2. Create a new APD
3. Select "MMIS IAPD" as the type
4. Fill out the rest of the form and press the Create an APD button
5. Within the developer tools, observe the GET call for the id of the APD that you just created. Look at the data in `apd.apdOverview.medicaidBusinessAreas` to see the subset of fields.

### Expected Behavior
The `medicaidBusinessAreas` field contains the full list of keys
```
    "medicaidBusinessAreas": {
      "waiverSupportSystems": false,
      "assetVerificationSystem": false,
      "claimsProcessing": true,
      "decisionSupportSystemDW": false,
      "electronicVisitVerification": false,
      "encounterProcessingSystemMCS": false,
      "financialManagement": true,
      "healthInformationExchange": false,
      "longTermServicesSupports": false,
      "memberManagement": false,
      "pharmacyBenefitManagementPOS": false,
      "programIntegrity": true,
      "providerManagement": false,
      "thirdPartyLiability": false,
      "other": false,
      "otherMedicaidBusinessAreas": ""
    }
```

### Actual Behavior
The `medicaidBusinessAreas` field contains a subset of the full list of keys. Seems to always be the same keys within the subset (example values):
```
{
    "claimsProcessing": true,
    "memberManagement": false,
    "programIntegrity": true,
    "providerManagement": true,
    "thirdPartyLiability": true,
    "other": true,
    "otherMedicaidBusinessAreas": "Something else"
}
```

### Dev Notes
- The POST call for the creation of the APD looks to be correctly sending the full list of `medicaidBusinessAreas` fields to the backend

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #4491 

### Description
Fixes issue where certain Medicaid Business Areas were not being persisted in the backend/db. 

### Significant changes or possible side effects
* Minor change to the backend model

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A HITECH APD type selected | creating a new APD | all appropriate options are displayed correctly with the correct values | jest (snapshot) |
| A MMIS APD type selected | creating a new APD | all appropriate options are displayed correctly with the correct values | jest (snapshot) |

### Steps to manually verify this change
1. Login as a user who can create APDs
2. Create a new APD
3. Select MMIS as the type
4. Select *all 16* of the Medicaid Business Areas (including filling out Other)
5. Before hitting the Create APD button, open DevTools and select the Network Tab
6. Check that the response of the APD GET (by it's ID) has all 16 of the options in apd -> apdOverview -> medicaidBusinessAreas
Screenshot 2023-02-14 at 8 42 33 AM

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: api/models/mmis.js [hunk 1]
```diff
@@ -16,17 +16,17 @@ const mmisSchema = new mongoose.Schema(
         asNeededUpdate: Boolean
       },
       medicaidBusinessAreas: {
-        waiverSupportSystems: Boolean,
-        assetVerificationSystem: Boolean,
+        waiverSupport: Boolean,
```
Comment (thetif) [inline api/models/mmis.js:19]: you might want to make the change the other way around, because you're going to need a migration if you change the model

**Review by thetif — APPROVED** [review]

File: api/models/mmis.js (hunk 1 above)
Comment (tbolt) [reply api/models/mmis.js:19]: sounds good. I should have thought of that 🤦

**Review by thetif — APPROVED** [review]

**Review by amyd11 — APPROVED** [review]
Nice job figuring out what the problem was

**Review by mirano-darren — APPROVED** [review]
fixed! well done!

**Comment by stephanieboydcms** [comment]
@tbolt I am good to proceed.  Thank you!

**Comment by stephanieboydcms** [comment]
@tbolt I am good to proceed.  Thank you!

**Review by stephanieboydcms — APPROVED** [review]
Good to proceed.  Thanks Ty

**Pull request merged**